components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -433,7 +433,22 @@ class PageWallet extends React.Component<Props, State> {
     return getLocale('greetingsVerified', { name: externalWallet.userName })
   }
 
+  disconnectWalletAlert = (): AlertWallet | null => {
```

Comment:
Can we do this inside of the `walletAlerts` function, similar to how it's done for some other alert types? I'm hoping that if we do that, we won't need the new `disconnectWalletAlert` prop on `WalletWrapper`, and hopefully we won't need to make any changes to `WalletWrapper` at all.

I'm thinking we'll want to add an `onAlertClose` method to the returned object that will clear the alert, maybe something like:

```js
onAlertClose: () => {
  this.actions.onClearAlert('showDisconnectWalletError')
}
```